### PR TITLE
ariaPressed name changed to prevent conflict with html attribute #235

### PR DESCRIPTION
**Pull request by rmenner**
# Alaska Airlines Pull Request

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

## Summary by Sourcery

Rename 'ariaPressed' to 'ariapressed' to prevent conflict with the HTML attribute and enhance demo HTML files by removing duplicate stylesheet links and deferring module script loading.

Bug Fixes:
- Rename 'ariaPressed' to 'ariapressed' to prevent conflict with the HTML attribute.

Enhancements:
- Remove duplicate stylesheet link in demo HTML files.
- Defer loading of module scripts in demo HTML files for improved performance.

### CODE REVIEW

File: src/component-base.mjs [hunk 1]
```diff
@@ -33,17 +33,17 @@ export default class ComponentBase extends LitElement {
     /**
      * @private
      */
-    this.ariaPressed = 'false';
+    this.ariapressed = 'false';
```
Comment (sun-mota) [inline src/component-base.mjs:36]: `ariaPressedState(ariaPressed)` function should reflect this change

**Review by sun-mota — CHANGES_REQUESTED** [review]
.

File: src/component-base.mjs (hunk 1 above)
Comment (rmenner) [reply src/component-base.mjs:36]: Maybe the one used in that function should be renamed. It is not related to the class property `this.ariapressed`

Comment (rmenner) [reply src/component-base.mjs:36]: The function arg is not really even used in the function it is pass into. It just gets passed in and returned right back out without an mutations. Going to make a issue to explore a refactor later. 

```
ariaPressedState(ariaPressed) {
    const ariaToggle = function (event) {
      const ariaPressedNode = this.shadowRoot.querySelector('[aria-pressed]');
      ariaPressedNode.setAttribute("aria-pressed", 'false');

      if (event.type === 'mousedown') {
        ariaPressedNode.ariaPressed = true;
      } else {
        ariaPressedNode.ariaPressed = false;
      }

      if (event.type === 'keydown') {
        if (event.code === 'Enter' || event.code === 'Space') {
          ariaPressedNode.ariaPressed = true;
        } else {
          ariaPressedNode.ariaPressed = false;
        }
      }
    };

    // Add event listeners
    this.addEventListener('mousedown', ariaToggle);
    this.addEventListener('mouseup', ariaToggle);
    this.addEventListener('keydown', ariaToggle);
    this.addEventListener('keyup', ariaToggle);

    return ariaPressed;
  }
```

**Review by sun-mota — APPROVED** [review]

**Pull request merged**